ta/avb/entry.c

```diff
@@ -336,6 +337,107 @@ static TEE_Result read_persist_value(uint32_t pt,
 
 	return res;
 }
+static TEE_Result provision_public_key(uint32_t pt,
+				       TEE_Param params[TEE_NUM_PARAMS])
+{
+	const uint32_t exp_pt = TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INPUT,
+						TEE_PARAM_TYPE_NONE,
+						TEE_PARAM_TYPE_NONE,
+						TEE_PARAM_TYPE_NONE);
+	TEE_Result res = TEE_SUCCESS;
+	TEE_ObjectHandle h = TEE_HANDLE_NULL;
+	uint32_t flags = TEE_DATA_FLAG_ACCESS_READ |
+			 TEE_DATA_FLAG_ACCESS_WRITE;
+
+	if (pt != exp_pt)
+		return TEE_ERROR_BAD_PARAMETERS;
+
+	char *pkey_to_provision = params[0].memref.buffer;
+	uint32_t pkey_to_provision_sz = params[0].memref.size;
+
+	res = TEE_OpenPersistentObject(storageid, pkey_obj_name,
```

Comment:
This is not needed, if the overwrite flag isn't provided `TEE_CreatePersistentObject()` will not overwrite the object.

Reply:
ok, will remove

Reply:
removed